src/filters/ff_enc.c

```diff
@@ -496,6 +497,16 @@ static GF_Err ffenc_process_video(GF_Filter *filter, struct _gf_ffenc_ctx *ctx)
 		force_intra = 2;
 	}
 
+	//don't repeat IDR sync if we already forced one
+	if (force_intra==2) {
+		if (ctx->prop_induced_reconfig) {
+			force_intra = 1;
+			ctx->prop_induced_reconfig = GF_FALSE;
+		} else {
+			ctx->prop_induced_reconfig = GF_TRUE;
+		}
+	}
+
```

Comment:
nice catch - can you do a separate PR for this one ?

Reply:
Done in #2888